File: Tutorials/10-Days-of-Javascript/Day-01/02-data-types.js
/**
 * Variables named firstInteger, firstDecimal, and firstString are declared for you.
 * You must use the + operator to perform the following sequence of operations:
 * 1 - Convert secondInteger to an integer (Number type), then sum it with firstInteger
 *     and print the result on a new line using console.log.
 * 2 - Convert secondDecimal to a floating-point number (Number type), then sum it with firstDecimal
 *     and print the result on a new line using console.log.
 * 3 - Print the concatenation of firstString and secondString on a new line using console.log.
 *      Note that must be printed first.
 */

function performOperation(secondInteger, secondDecimal, secondString) {
  // Declare a variable named 'firstInteger' and initialize with integer value 4.
  const firstInteger = 4;
  // Declare a variable named 'firstDecimal' and initialize with floating-point value 4.0.
  const firstDecimal = 4.0;
  // Declare a variable named 'firstString' and initialize with the string "HackerRank".
  const firstString = 'HackerRank ';

  // Write code that uses console.log to print the sum of the 'firstInteger' and 'secondInteger'
  // (converted to a Number Int type) on a new line.
  const secondInt = parseInt(secondInteger);
  const sumInt = firstInteger + secondInt;
  console.log(`Integer ➡ ${sumInt}`);

  // Write code that uses console.log to print the sum of 'firstDecimal' and 'secondDecimal'
  // (converted to a Number Float type) on a new line.
  const secondDec = parseFloat(secondDecimal);
  const sumDec = firstDecimal + secondDec;
  console.log(`Decimal ➡ ${sumDec}`);

  // Write code that uses console.log to print the concatenation of 'firstString' and 'secondString'
  console.log(`String ➡ ${firstString + secondString}`);
}

// Output
performOperation(3, 2.5, '🚀');

File: Tutorials/30-Days-of-Code/02-data-types.py
'''
Task:
  Complete the code in the editor below. The variables i, d, and s
  are already declared and initialized for you. You must:
  1 - Declare 3 variables: one of type int, one of type double, and one of type String.
  2 - Read 3 lines of input from stdin (according to the sequence
      given in the Input Format section below) and initialize your 3 variables.
  3 - Use the + operator to perform the following operations:
    a - Print the sum of i plus your int variable on a new line.
    b - Print the sum of d plus your double variable to a scale of one decimal place on a new line.
    c - Concatenate s with the string you read as input and print the result on a new line.
'''

# Solution
i = 4
d = 4.0
s = 'HackerRank '
# Declare second integer, double, and String variables.
int_2 = input('Enter an integer number: ')
double_2 = input('Enter a decimal number: ')
string_2 = input('Enter a cool word: ')
# Read and save an integer, double, and String to your variables.
i_2 = int(int_2)
d_2 = float(double_2)
s_2 = string_2
# Print the sum of both integer variables on a new line.
print(f'➡ {i + i_2}')
# Print the sum of the double variables on a new line.
print(f'➡ {d + d_2}')
# Concatenate and print the String variables on a new line
# The 's' variable above should be printed first.
print(f'➡ {s + s_2}')
